src/assets/scripts/client/aircraft/StripView/StripViewController.js

```diff
@@ -149,7 +158,7 @@ export default class StripViewController {
             let stripViewModel = this._collection.findStripByAircraftId(aircraftModel.id);
             
             if (typeof stripViewModel === "undefined") {
-                stripViewModel = this.createStripView(aircraftModel);
+                stripViewModel = this.createStripView(aircraftModel, aircraftModel.isDeparture());
```

Comment:
the `aircraft.isDeparture()` method is unnecessary here.  since `.createStripView()` is receiving the entire `AircraftModel`, you don't need to pass it separately.  the receiving method call it directly.